functions/setupProject.js

```diff
@@ -1,54 +1,152 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable consistent-return */
 /* eslint-disable import/no-dynamic-require */
 /* eslint-disable global-require */
 const admin = require('firebase-admin');
-const readline = require('readline');
+const inquirer = require('inquirer');
+const fs = require('fs');
 
-const rl = readline.createInterface({
-  input: process.stdin,
-  output: process.stdout
-});
+const configPath = '../src/state/api/index.js';
 
-rl.question('Enter the path to the service account key file: ', path => {
-  const serviceAccount = require(path);
+const importPath = '../src/firebase.js';
+
+const questions = [
+  {
+    type: 'input',
+    name: 'path',
+    message: 'Enter the path to the service account key file: ',
+  },
+  {
+    type: 'input',
+    name: 'databaseURL',
+    message: 'Enter database URL: ',
+  },
+  {
+    type: 'input',
+    name: 'email',
+    message: 'Enter user email: ',
+  },
+  {
+    type: 'password',
+    name: 'password',
+    message: 'Enter user password: ',
+    mask: '*',
+  },
+  {
+    type: 'list',
+    name: 'database',
+    message: 'Select the database of your choice:',
+    choices: ['Realtime Database', 'Firestore'],
+  },
```

Comment:
Let's add a new question asking if we want to delete either 'Realtime Database' or 'Firestore' cloud functions. If the user selects 'No', then we're not going to delete the cloud functions, but if they select 'Yes', we delete unused cloud functions.